src/test/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformerTest.java

```diff
@@ -2516,17 +2529,171 @@ public void givenNullFormWithOcrFormType_thenThrowNoError() {
 
         ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
             null).build();
+
         given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+
+        assertTrue(result.getErrors().size() == 1);
+        assertTrue(result.getWarnings().isEmpty());
+    }
+
+    @Test
+    public void givenValidFormAndWithNullFormTypeInput_thenNoThrowError() {
+        prepareData(null);
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            "sscs5").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
         CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
 
+
         assertTrue(result.getErrors().isEmpty());
         assertTrue(result.getWarnings().isEmpty());
     }
 
+    @Test
+    public void givenOtherFormAndWithNullFormTypeInput_thenThrowError() {
+        prepareData(null);
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            "Other").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(ScannedData.builder().ocrCaseData(pairs).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+
+        assertTrue(result.getErrors().size() == 1);
+        assertTrue(result.getWarnings().isEmpty());
+    }
+
+    @SuppressWarnings("unchecked")
+    @Test
+    public void givenOtherFormAndWithSscs2FormTypeInput_thenDocumentUpdated() {
+        prepareData("SSCS2");
+
+        List<InputScannedDoc> records = new ArrayList<>();
+        InputScannedDoc scannedRecord = buildTestScannedRecord(DocumentLink.builder().documentUrl("www.test.com").build(), "Other");
+        records.add(scannedRecord);
+
+        ExceptionRecord exceptionRecord = ExceptionRecord.builder().ocrDataFields(ocrList).id(null).exceptionRecordId("123456").formType(
+            "Other").build();
+
+        given(sscsJsonExtractor.extractJson(exceptionRecord)).willReturn(
+            ScannedData.builder().ocrCaseData(pairs).records(records).openingDate(LocalDateTime.now().toLocalDate().toString()).build());
+
+        CaseResponse result = transformer2.transformExceptionRecord(exceptionRecord, false);
+
+        Object resultObject = result.getTransformedCase().get("sscsDocument");
+        Assert.isInstanceOf(List.class, resultObject);
+
+        Assertions.assertThat((List<SscsDocument>)resultObject)
+            .extracting(SscsDocument::getValue)
+            .extracting(SscsDocumentDetails::getDocumentType)
+            .contains("sscs2");
```

Comment:
These last three tests are all very similar and can also be extracted into a common assertion helper function.
Args would be String formType and String expected.